G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelSave/ShouldSaveExcelFile_WhenSaveCommandIsCalled.txt

```diff
@@ -0,0 +1,9 @@
+﻿♥filepath = ‴♥environment⟦TEMP⟧\tempexcelfile.xlsx‴
+
+excel.open
+excel.setvalue value ‴Test text.‴ row 1 colname ‴A‴
+excel.setvalue value ‴54642 5854 22‴ row 1 colname ‴B‴
+excel.save path ♥filepath
+excel.close
+file.exists filename ♥filepath
```

Comment:
I guess this is the test - the file exists or this line will throw an exception.
It would be nice to check that the file is not empty, or even better, start again excel and read the values, otherwise storing them (lines 7 & 8) doesn't make sense.